src/models/validate.js

```diff
@@ -63,3 +89,8 @@ export const validateModel = (data, model, options) => {
 }
 
 export default validateModel
+
+/** require Yes or No */
+export const hasYesOrNo = {
```

Comment:
Is there a reason why `hasYesOrNo` is in `validate.js` rather than broken out into a different file like the rest of the validators?

Reply:
@christopherhuii everything that's in the `src/models/validators` is a custom validator function that gets initialized in this `validate.js` file. `hasYesOrNo` is just a shorthand for this `inclusion` rule, since it is pretty common. I wasn't sold on how to organize something like that so I just stuck it here for now 🤷‍♀